src/shared/tpm2-util.c

```diff
@@ -1971,8 +2229,9 @@ int tpm2_extend_bytes(
                                 sym_Tss2_RC_Decode(rc));
 
         return 0;
-#else /* HAVE_OPENSSL */
```

Comment:
why this  change to remove the comment and add newline?

Reply:
probably a rebase issue, let me look again.

Reply:
rebase glitch, messed up a few of the hunks, fixed.